package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Election-Day-App",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./server/index"
+  },
+  "dependencies": {
+    "cookie-parser": "~1.4.3",
+    "express": "~4.16.0"
+  },
+  "devDependencies": {
+    "debug": "~2.6.9",
+    "eslint": "^5.5.0",
```

Comment:
Great that you added eslint but where is the configuration file (`.eslintrc`). This file should be in the root folder of your project and should contained some configuration for basic rules your team agrees with.